src/GovUk.Education.ExploreEducationStatistics.Data.Model/LocationAttribute.cs

```diff
@@ -25,13 +26,48 @@ public virtual string GetCodeOrFallback()
     }
 
     public GeographicLevel GeographicLevel => Enum.Parse<GeographicLevel>(GetType().Name);
+
+    public abstract KeyValuePair<string, string>[] CsvValues { get; }
+
+    // The lower the number, the sooner it should appear in the CSV.
+    public abstract int CsvPriority { get; }
```

Comment:
Not too keen on this. The current implementation is using inheritance so that we can get feedback at compile time, ensuring that we're filling out any new implementations of `LocationAttribute` correctly.

Given how anaemic the domain model is anyway, it doesn't really seem like a problem to put the CSV functionality in here.

I think if we need to add more functionality to this in the future, we can reassess this, but I'd err on the side of KISS.